website/source/docs/providers/aws/d/availability_zones.html.markdown

```diff
@@ -35,10 +35,15 @@ resource "aws_subnet" "secondary" {
 
 ## Argument Reference
 
-There are no arguments for this data source.
+The following arguments are supported:
+
+* `state` - (Optional) Allows to filter list of Availability Zones based on their
+current state. Can be either `"available"`, `"information"`, `"impaired"` or
+`"unavailable"`. By default the list includes a complete set of Availability Zones
+to which the underlying AWS account has access, regardless of their state.
 
 ## Attributes Reference
 
 The following attributes are exported:
 
-* `names` - A list of the availability zone names available to the account.
+* `names` - A list of the Availability Zone names available to the account.
```

Comment:
@apparentlymart we can merge this one for now and then follow up on the docs later :)